### PR TITLE
New Azure Storage Playbooks

### PR DESCRIPTION
**Pull request by toddm92**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
New Prisma Cloud Azure Storage Playbooks

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Comment by toddm92** [comment]
@mayagoldb ..last two Azure (Storage) playbooks.

File: Packs/PrismaCloud/ReleaseNotes/1_8_0.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+- Updated classifier for new Azure Storage playbooks
+##### Prisma Cloud - Classifier
+- Updated classifier for new Azure Storage playbooks
+##### RedLock
+- Updated for new Azure Storage playbooks
+##### prismaCloud_app
+- Updated for new Azure Storage playbooks
+
+#### Incident Fields
+- **Subscription Updated On**
```
Comment (tomneeman151293) [inline Packs/PrismaCloud/ReleaseNotes/1_8_0.md:13]:
```suggestion
- **Subscription Updated On** - Added **Azure Storage Misconfiguration** incident to **Subscription Updated On** Associated types.
```

File: Packs/PrismaCloud/ReleaseNotes/1_8_0.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+- **Subscription Description**
```
Comment (tomneeman151293) [inline Packs/PrismaCloud/ReleaseNotes/1_8_0.md:14]:
```suggestion
- **Subscription Description** - Added **Azure Storage Misconfiguration** incident to **Subscription Description** Associated types.
```

File: Packs/PrismaCloud/ReleaseNotes/1_8_0.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+- **Resource API Name**
```
Comment (tomneeman151293) [inline Packs/PrismaCloud/ReleaseNotes/1_8_0.md:15]:
```suggestion
- **Resource API Name** - Added **Azure Storage Misconfiguration** incident to **Resource API Name** Associated types.
```

File: Packs/PrismaCloud/ReleaseNotes/1_8_0.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+- **Subscription Created On**
```
Comment (tomneeman151293) [inline Packs/PrismaCloud/ReleaseNotes/1_8_0.md:16]:
```suggestion
- **Subscription Created On** - Added **Azure Storage Misconfiguration** incident to **Subscription Created On** Associated types.
```

File: Packs/PrismaCloud/ReleaseNotes/1_8_0.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+- **Subscription Updated By**
```
Comment (tomneeman151293) [inline Packs/PrismaCloud/ReleaseNotes/1_8_0.md:17]:
```suggestion
- **Subscription Updated By** - Added **Azure Storage Misconfiguration** incident to **Subscription Updated By** Associated types.
```

File: Packs/PrismaCloud/ReleaseNotes/1_8_0.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+- **Prisma Cloud Status**
```
Comment (tomneeman151293) [inline Packs/PrismaCloud/ReleaseNotes/1_8_0.md:18]: - **Prisma Cloud Status** - Added **Azure Storage Misconfiguration** incident to **Prisma Cloud Status** Associated types.

File: Packs/PrismaCloud/ReleaseNotes/1_8_0.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+- **Subscription Created By**
```
Comment (tomneeman151293) [inline Packs/PrismaCloud/ReleaseNotes/1_8_0.md:19]:
```suggestion
- **Subscription Created By** - Added **Azure Storage Misconfiguration** incident to **Subscription Created By** Associated types.
```

File: Packs/PrismaCloud/ReleaseNotes/1_8_0.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+- **Prisma Cloud Time**
```
Comment (tomneeman151293) [inline Packs/PrismaCloud/ReleaseNotes/1_8_0.md:20]:
```suggestion
- **Prisma Cloud Time** - Added **Azure Storage Misconfiguration** incident to **Prisma Cloud Time** Associated types.
```
```

File: Packs/PrismaCloud/ReleaseNotes/1_8_0.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+- **VPC ID**
```
Comment (tomneeman151293) [inline Packs/PrismaCloud/ReleaseNotes/1_8_0.md:21]:
```suggestion
- **VPC ID** - Added **Azure Storage Misconfiguration** incident to **VPC ID** Associated types.
```

File: Packs/PrismaCloud/ReleaseNotes/1_8_0.md [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+- **Prisma Cloud ID**
```
Comment (tomneeman151293) [inline Packs/PrismaCloud/ReleaseNotes/1_8_0.md:22]:
```suggestion
- **Prisma Cloud ID** - Added **Azure Storage Misconfiguration** incident to **Prisma Cloud ID** Associated types.
```

File: Packs/PrismaCloud/ReleaseNotes/1_8_0.md [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+- **Subscription Type**
```
Comment (tomneeman151293) [inline Packs/PrismaCloud/ReleaseNotes/1_8_0.md:23]:
```suggestion
- **Subscription Type** - Added **Azure Storage Misconfiguration** incident to **Subscription Type** Associated types.
```

File: Packs/PrismaCloud/ReleaseNotes/1_8_0.md [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+- **Resource Cloud Type**
```
Comment (tomneeman151293) [inline Packs/PrismaCloud/ReleaseNotes/1_8_0.md:24]:
```suggestion
- **Resource Cloud Type** - Added **Azure Storage Misconfiguration** incident to **Resource Cloud Type** Associated types.
```

File: Packs/PrismaCloud/ReleaseNotes/1_8_0.md [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+- **Subscription Name**
```
Comment (tomneeman151293) [inline Packs/PrismaCloud/ReleaseNotes/1_8_0.md:25]:
```suggestion
- **Subscription Name** - Added **Azure Storage Misconfiguration** incident to **Subscription Name** Associated types.
```

File: Packs/PrismaCloud/ReleaseNotes/1_8_0.md [hunk 14]
```diff
@@ -0,0 +1,58 @@
...
+- **RRN**
```
Comment (tomneeman151293) [inline Packs/PrismaCloud/ReleaseNotes/1_8_0.md:26]:
```suggestion
- **Subscription Assigned By** - Added **Azure Storage Misconfiguration** incident to **Subscription Assigned By** Associated types.
```

File: Packs/PrismaCloud/ReleaseNotes/1_8_0.md [hunk 15]
```diff
@@ -0,0 +1,58 @@
...
+- **Subscription Assigned By**
```
Comment (tomneeman151293) [inline Packs/PrismaCloud/ReleaseNotes/1_8_0.md:27]:
```suggestion
- **Subscription Assigned By** - Added **Azure Storage Misconfiguration** incident to **Subscription Assigned By** Associated types.
```

File: Packs/PrismaCloud/ReleaseNotes/1_8_0.md [hunk 16]
```diff
@@ -0,0 +1,58 @@
...
+- **Subscription ID**
```
Comment (tomneeman151293) [inline Packs/PrismaCloud/ReleaseNotes/1_8_0.md:28]:
```suggestion
- **Subscription ID** - Added **Azure Storage Misconfiguration** incident to **Subscription ID** Associated types.
```

File: Packs/PrismaCloud/ReleaseNotes/1_8_0.md [hunk 17]
```diff
@@ -0,0 +1,58 @@
...
+- **Prisma Cloud Reason**
```
Comment (tomneeman151293) [inline Packs/PrismaCloud/ReleaseNotes/1_8_0.md:29]:
```suggestion
- **Prisma Cloud Reason** - Added **Azure Storage Misconfiguration** incident to **Prisma Cloud Reason** Associated types.
```

File: Packs/PrismaCloud/ReleaseNotes/1_8_0.md [hunk 18]
```diff
@@ -0,0 +1,58 @@
...
+- **System Default**
+- **Prisma Cloud Rules**
```
Comment (tomneeman151293) [inline Packs/PrismaCloud/ReleaseNotes/1_8_0.md:31]:
```suggestion
- **Prisma Cloud Rules** - Added **Azure Storage Misconfiguration** incident to **Prisma Cloud Rules** Associated types.
```

File: Packs/PrismaCloud/ReleaseNotes/1_8_0.md [hunk 19]
```diff
@@ -0,0 +1,58 @@
...
+
+#### Incident Types
+- **Azure Storage Misconfiguration**
```
Comment (tomneeman151293) [inline Packs/PrismaCloud/ReleaseNotes/1_8_0.md:34]:
```suggestion
- **Azure Storage Misconfiguration** - New incident type.
```

File: Packs/PrismaCloud/ReleaseNotes/1_8_0.md [hunk 20]
```diff
@@ -0,0 +1,58 @@
...
+
+#### Layouts
+##### New: Azure Storage Misconfiguration Incident
+- Updated for new Azure Storage playbooks
```
Comment (tomneeman151293) [inline Packs/PrismaCloud/ReleaseNotes/1_8_0.md:38]:
```suggestion
- New layout for Azure Storage Misconfiguration Incident.
```

**Review by tomneeman151293 — CHANGES_REQUESTED** [review]
Thanks for the contribution! Added a few requested changes in the release notes. If you could go over them.
Also, the Layout file you added is available from 6.0.0 and above, I will add a layout supporting versions below 6.0.0 to your branch as the playbook is supported from 5.0.0

File: Packs/PrismaCloud/Layouts/layoutscontainer-Azure_Storage_Misconfiguration.json [hunk 21]
```diff
@@ -0,0 +1,642 @@
+{
+    "id": "Azure Storage Misconfiguration Incident",
```
Comment (tomneeman151293) [inline Packs/PrismaCloud/Layouts/layoutscontainer-Azure_Storage_Misconfiguration.json:2]:
```suggestion
    "id": "Azure Storage Misconfiguration",
```
ID of layout should equal to the incident type it is related to

File: Packs/PrismaCloud/Layouts/layoutscontainer-Azure_Storage_Misconfiguration.json [hunk 22]
```diff
@@ -0,0 +1,642 @@
...
+    "group": "incident",
+    "name": "Azure Storage Misconfiguration Incident",
```
Comment (tomneeman151293) [inline Packs/PrismaCloud/Layouts/layoutscontainer-Azure_Storage_Misconfiguration.json:4]:
```suggestion
    "name": "Azure Storage Misconfiguration",
```

**Review by tomneeman151293 — CHANGES_REQUESTED** [review]
Changes to layout container to match the new incident type added, also here is the zip of layout file support below 6.0.0. Please extract this zip and and add the file to layouts directory.
[layout-detailsV2-Azure_Storage_Misconfiguration.json.zip](https://github.com/demisto/content/files/6748828/layout-detailsV2-Azure_Storage_Misconfiguration.json.zip)

**Comment by toddm92** [comment]
@tomneeman151293 suggestions resolved.  Thx.

**Comment by toddm92** [comment]
@tomneeman151293 new layout file added

File: Packs/PrismaCloud/Layouts/layout-detailsV2-Azure_Storage_Misconfiguration.json [hunk 23]
```diff
@@ -0,0 +1,646 @@
+{
+  "fromVersion": "4.1.0",
```
Comment (tomneeman151293) [inline Packs/PrismaCloud/Layouts/layout-detailsV2-Azure_Storage_Misconfiguration.json:2]: This should fix the validation failure
```suggestion
  "fromVersion": "5.0.0",
```

**Review by tomneeman151293 — CHANGES_REQUESTED** [review]
Thanks! Looks great!
Added a small fix for the newly added layout (my bad) to fix validation failure.
Please commit and re-run to see if validation passes

**Comment by toddm92** [comment]
> Thanks! Looks great!
> Added a small fix for the newly added layout (my bad) to fix validation failure.
> Please commit and re-run to see if validation passes

@tomneeman151293  done.

**Comment by tomneeman151293** [comment]
@mayagoldb I am done with the review from my side, waiting for your side as well

**Comment by toddm92** [comment]
@tomneeman151293 I believe @mayagoldb is out of the office until the end of July.  If we need two reviewers, I guess we'll have to wait.  Otherwise, we can push this one thru.

**Comment by tomneeman151293** [comment]
@Michal-Vardi @Shellyber Any way we can push this forward?

**Review by mayagoldb — APPROVED** [review]

**Review by tomneeman151293 — APPROVED** [review]

**Pull request merged**